commands/pubsub-shardchannels.md

```diff
@@ -0,0 +1,19 @@
+Lists the currently *active shard channels*.
+
+An active shard channel is a Pub/Sub shard channel with one or more subscribers.
+
+If no `pattern` is specified, all the channels are listed, otherwise if pattern is specified only channels matching the specified glob-style pattern are listed.
+
+Cluster note: `PUBSUB`'s replies in a cluster only report information from the node's Pub/Sub context, rather than the entire cluster.
```

Comment:
I don't know what this is trying to say. I see it in the corresponding pubsub-channels info section, but it makes more sense there since pubsub is a global scope object. I would suggest dropping this.

Reply:
"The information we return about the active shard channels are at the shard level and not at the cluster level."

Do you think we should put the above then?

Reply:
That sounds reasonable, I understand what you are trying to communicate.